tardis/plasma/base.py

```diff
@@ -38,15 +40,17 @@ def __getattr__(self, item):
     def __setattr__(self, key, value):
         if key != "module_dict" and key in self.outputs_dict:
             raise AttributeError(
-                "Plasma inputs can only be updated using " "the 'update' method"
+                "Plasma inputs can only be updated using " 
+                "the 'update' method"
             )
         else:
             super(BasePlasma, self).__setattr__(key, value)
 
     def __dir__(self):
         attrs = [item for item in self.__dict__ if not item.startswith("_")]
+        class_dict = self.__class__.__dict__
```

Comment:
why do we need to assign this to a different variable?

Reply:
This was in an attempt to follow the PEP8 guidelines in an attempt to prevent the black check from failing. Since it failed anyway, I will simply reverse this change

Reply:
sounds good. you can do line breaks